curriculum/challenges/english/08-coding-interview-prep/rosetta-code/self-describing-numbers.english.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Self Describing Numbers
+id: 5eaf48389ee512d4d103684b
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+There are several so-called "self describing" or <a href="https://en.wikipedia.org/wiki/Self-descriptive_number">"self-descriptive"</a> integers. <br />
+
+An integer is said to be "self-describing" if it has the property that, when digit positions are labeled 0 to N-1, the digit in each postion is eqal to the number of times that that digit appears in the number. <br />
```

Comment:
Couple of minor typos and we can remove the line break:

`An integer is said to be "self-describing" if it has the property that, when digit positions are labeled 0 to N-1, the digit in each position is equal to the number of times that that digit appears in the number.`